boards/arm/npcx7m6fb_evb/npcx7m6fb_evb.dts

```diff
@@ -36,7 +36,8 @@
 
 	def_lvol_io_list {
 		compatible = "nuvoton,npcx-lvolctrl-def";
-		lvol_io_pads = <>; /* Put low-voltage io pads here */
+		lvol_io_pads = <&lvol_io90   /* I2C1_SCL0 1.8V support */
```

Comment:
Is this just a demonstration, or is this something the board actually needs?  IOW could somebody put 3.3V devices on this bus, then have to remove this from the board devicetree to make things work?

Reply:
@pabigot :
It depends on what customers need. There're no default PU resistances on i2c's pads of npcx7m6fb's EVB. (Please refer bottom right corner of the [picture](https://docs.zephyrproject.org/latest/_images/npcx7m6fb_evb.png)). The PU resistance value influences i2c's bus frequency a lot, hence we don't put them on i2c pads of EVB by default. 

There're two cases for our i2c pads:
1. The bus voltage of i2c device is 3.3V. Since the origin detection level of i2c's pads is 3.3V (ie. Vih = 3.3V*0.7 and Vil = 3.3V *0.3), the customers needn't put these pads into "lvol_io_pads" array. They can connect 3.3 PU via the other circuit or our PU connectors which provide 3.3 PU on EVB.
2. The bus voltage of i2c device is 1.8V. The customers need to put these pads into "lvol_io_pads" array and their detection level changes to 1.8V. (ie. Vih = 1.8V * 0.7 and Vil = 1.8V  * 0.3). They can connect 1.8 PU via the other circuit or our PU connectors which provide 1.8V PU on EVB.

Reply:
So it sounds like the default configuration is 3.3V, and the board does not require that it be anything else.  In that case I don't think the board devicetree should be changing it to a non-default configuration as an example.  The instructions for changing PU levels should be somewhere other than the operational part of the devicetree.

I don't think it's a good idea, but since the board is not available to casual users who could be asking questions on Zephyr support forums, if you want to change the behavior as an example I wouldn't block it.  You should probably add a comment though saying that it's not necessarily correct.  Or put the properties inside such a comment.

Reply:
Thanks for the suggestion. I will put this example into a comment before this property.